modules/add_takeoff_and_landing_command.py

```diff
@@ -59,13 +59,13 @@ def add_takeoff_and_landing_command(commands: "list[dronekit.Command]",
         0,
         0,
         0,  # param1
-        0,
-        0,
+        2,
+        None,
         0,
         0,
         0,
         0,
     )
     commands.append(landing_command)
 
-    return True, commands
+    return True, commands
```

Comment:
Add an extra line at the bottom of the file. If you see this red mark here it means you must add a line at the end of the file.